modules/contact/test/tests/bouncing-block-contact/variational-frictional.i

```diff
@@ -0,0 +1,196 @@
+    type = ConstantIC
+  []
+[]
+
+[Kernels]
+  [disp_x]
+    type = MatDiffusion
+    variable = disp_x
+  []
+  [disp_y]
+    type = MatDiffusion
+    variable = disp_y
+  []
+[]
+
+[AuxVariables]
+  [procid]
+    family = MONOMIAL
+    order = CONSTANT
+  []
+[]
+
+[AuxKernels]
+  [procid]
+    type = ProcessorIDAux
+    variable = procid
+  []
+[]
+
+[Constraints]
+  [frictional_normal_lm]
+    type = ComputeFrictionalForceLMMechanicalContact
+    primary_boundary = 10
+    secondary_boundary = 20
+    primary_subdomain = 3
+    secondary_subdomain = 4
+    variable = frictional_normal_lm
+    friction_lm = frictional_tangential_lm
+    disp_x = disp_x
+    disp_y = disp_y
+    mu = 0.1
+    normalize_c = true
+    c = 1.0e-2
+    c_t = 1.0e-1
+  []
+  [normal_x]
+    type = NormalMortarMechanicalContact
+    primary_boundary = 10
+    secondary_boundary = 20
+    primary_subdomain = 3
+    secondary_subdomain = 4
+    variable = frictional_normal_lm
+    secondary_variable = disp_x
+    component = x
+    use_displaced_mesh = true
+    compute_lm_residuals = false
+  []
+  [normal_y]
+    type = NormalMortarMechanicalContact
+    primary_boundary = 10
+    secondary_boundary = 20
+    primary_subdomain = 3
+    secondary_subdomain = 4
+    variable = frictional_normal_lm
+    secondary_variable = disp_y
+    component = y
+    use_displaced_mesh = true
+    compute_lm_residuals = false
+  []
+  [tangential_x]
+    type = TangentialMortarMechanicalContact
+    primary_boundary = 10
+    secondary_boundary = 20
+    primary_subdomain = 3
+    secondary_subdomain = 4
+    variable = frictional_tangential_lm
+    secondary_variable = disp_x
+    component = x
+    use_displaced_mesh = true
+    compute_lm_residuals = false
+  []
+  [tangential_y]
+    type = TangentialMortarMechanicalContact
+    primary_boundary = 10
+    secondary_boundary = 20
+    primary_subdomain = 3
+    secondary_subdomain = 4
+    variable = frictional_tangential_lm
+    secondary_variable = disp_y
+    component = y
+    use_displaced_mesh = true
+    compute_lm_residuals = false
+  []
+[]
+
+[BCs]
+  [botx]
+    type = DirichletBC
+    variable = disp_x
+    boundary = 40
+    value = 0.0
+  []
+  [boty]
+    type = DirichletBC
+    variable = disp_y
+    boundary = 40
+    value = 0.0
+  []
+  [topy]
+    type = FunctionDirichletBC
+    variable = disp_y
+    boundary = 30
+    function = '${starting_point} * cos(2 * pi / 40 * t) + ${offset}'
+  []
+  [leftx]
+    type = FunctionDirichletBC
+    variable = disp_x
+    boundary = 50
+    function = '1e-2 * t'
+  []
+[]
+
+[Executioner]
+  type = Transient
+  end_time = 200
+  dt = 5
+  dtmin = .1
+  solve_type = 'PJFNK'
+  petsc_options = '-snes_converged_reason -ksp_converged_reason'
+  petsc_options_iname = '-pc_type -pc_factor_shift_type -pc_factor_shift_amount'
+  petsc_options_value = 'lu       NONZERO               1e-15'
+  l_max_its = 30
+  nl_max_its = 25
+  line_search = 'none'
+  snesmf_reuse_base = false
```

Comment:
I believe there is a way (IIRC) to set this parameter within the code. If that's possible, we may want to set it within the mortar constraints --eventually?

Reply:
I think it may actually be unnecessary to set this to false for mortar. Or if it is necessary I don’t understand why haha. This should not be stateful like some node-face code, eg our residual should be entirely determined by the current solution. Like I know that if a node ping pongs a couple times in the node-face case we lock it in place which is an action independent of the current solution.

Reply:
I believe we started setting that flag to false when we were using `NormalNodalLMMechanicalContact` stuff (https://github.com/idaholab/moose/issues/16821) which partially relied on node-face machinery. I have been carried that setting even with the latest full-mortar contact objects, probably unnecessarily as you point out.

Reply:
Alright I removed this flag, and things passed locally. Hopefully it will pass on CIVET too